docs/wiki/The CI environment - Publishing.md

```diff
@@ -28,35 +28,38 @@ The publishing works as follows:
 
 1. The script `utilities/pipelines/resourcePublish/Get-ModulesToPublish.ps1` gets all changed module files including child modules and handles the logic of propagating the appropriate module version to be used:
    1. The major (`x.0`) and minor (`0.x`) version are set based on the file `version.json` in the module folder.
-   2. The patch (`0.0.x`) version is calculated based on the number of commits on the `HEAD` ref (aka. git height). This will cause the patch version to never reset to 0 with major and/or minor increment, as specified for [semver](https://semver.org/).
-   3. The module is published with a patch specific version (`x.y.z`). For Template Specs and Bicep Registry a major (`x`) and minor (`x.x`) version is also updated, allowing a consumer to use the latest version of any major or minor version.
+   1. The patch (`0.0.x`) version is calculated based on the number of commits on the `HEAD` ref (aka. git height). This will cause the patch version to never reset to 0 with major and/or minor increment, as specified for [semver](https://semver.org/).
+   1. The module is published with a patch specific version (`x.y.z`). For Template Specs and Bicep Registry only, absolute major (`x`) and minor (`x.y`) versions are also updated, allowing a consumer to:
+      - Reference the latest version of a major, i.e. the latest minor and patch of an absolute major version.
```

Comment:
What's an `absolute` version?

Reply:
I'd suggest to give an example each

Reply:
We have a specific section below: Example usage. I think it would make sense to revise it to cover 1:1 examples (with values) of each point covered by the how it works paragraph. Maybe @MariusStorhaug can help?

Reply:
Agreed. I have to admit I did not undersand the meaning of 'absolute' when I read it :D

Reply:
I agree it's not correct either. I'm having a hard time to find a clear way to explain it.

Reply:
Please check if better now

Reply:
I just really dislike the `absolute` an my guess is we hit a dead end and should rethink the paragraph

Reply:
Let's see what Norway has to say

Reply:
I removed the absolute in my last commit